### PR TITLE
docs(cloud): document trigger and list cloud functionality

### PR DESCRIPTION
**Pull request by kkmlr**
## What this PR does / why we need it:
document trigger and list cloud functionality

## Does this PR introduce a user-facing change?
```
no
```

### CODE REVIEW

**Review by soerenmartius — DISMISSED** [review]
lgtm

**Comment by i4ki** [comment]
FYI: the additional statuses filters are still *unreleased*

File: docs/cli/cmdline/trigger.md [hunk 1]
```diff
@@ -15,19 +15,32 @@ next:
...
-Per default, triggers are managed in the `.tmtriggers` directory.
+The trigger mechanism has various use cases. It may be that a previous CI/CD run failed or that you have detected drift between the code and the actual state. For those using Terramate Cloud, the additional `--experimental-status=<status>` argument can be used to trigger stacks that are in one of the following states:
+
+| Status      | Meaning                                                          |
+| ----------- | ---------------------------------------------------------------- |
+| `ok`        | The stack is not drifted and the last deployment succeeded       |
+| `failed`    | The last deployment of the stack failed so the status is unknown |
+| `drifted`   | The actual state is different from that defined in code          |
+| `unhealthy` | The stack is drifted or failed                                   |
+| `healthy`   | The stack is ok                                                  |
```
[the remark this answers is not in the document]
Comment (kkmlr) [reply docs/cli/cmdline/trigger.md]: fixed. does that read better?

**Review by i4ki — APPROVED** [review]
LGTM

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**